collection-scripts/gather

```diff
@@ -7,7 +7,7 @@ resources=()
 resources+=(ns/openshift-cnv ns/kubevirt-hyperconverged ns/openshift-operator-lifecycle-manager ns/openshift-marketplace)
 
 # KubeVirt network related namespaces
-resources+=(ns/kubemacpool-system ns/linux-bridge ns/openshift-sdn ns/cluster-network-addons-operator ns/nmstate ns/sriov)
+resources+=(ns/cluster-network-addons ns/kubemacpool-system ns/linux-bridge ns/openshift-sdn ns/cluster-network-addons-operator ns/nmstate ns/sriov)
```

Comment:
In this case we do not need `ns/cluster-network-addons-operator`. Please remove.

Reply:
Don't we keep those for compatibility with older installations maybe?

BTW the must-gather is built so it works with standalone kubevirt+CNAO, U/S HCO and D/S, all at the same time?

Reply:
The goal is to have specific must-gather collection image for product version. We have nothing to be compatible with. It is the first team we release it.

It should work with specific CNV version for upstream and downstream. So far there was no need to do anything specific for d/s.

Reply:
If it is not to be used outside of HCO setup, we can drop kubemacpool-system, linux-bridge and nmstate. Also, replace sriov with sriov-network-operator.

Reply:
@mmirecki could you address the suggestions ?

Reply:
Afaik, HCO setup is the only deployment option and we do not worry about backwards compatilibity.
Dropping the old ns's.

Reply:
sriov done in a separate pr